### ISSUE TITLE
Parse conversations_history response

### ISSUE THREAD

**Issue by hallz11** [slack-ruby/slack-ruby-client#383]
Hi, I would like to know how to parse the response for `client.conversations_history` request. I usually use JSON.parse and loop the messages. But, I cannot do that for this case since the response looks like this:
```
channel_actions_count=0, channel_actions_ts=, has_more=true, messages=[#<Slack::Messages::Message attachments=#<Hashie::Array [#<Slack::Messages::Message actions=#<Hashie::Array [#<Slack::Messages::Message id="1" style="" text=""
....
```

Thank you.

**Comment by dblock** [comment]
I don't think you need to parse anything, the library does it for you. In [this code](https://github.com/dblock/slack-sup/blob/cb17133e496d6ae16a12b4fa837c9d77657fb12a/slack-sup/models/sup.rb#L128) I am just doing `.messages` and you get back a collection of `Slack::Messages::Message`. 

Does this help?

**Comment by hallz11** [comment]
Hi, yes it helps. Sorry my bad not noticing that. Thank you for the help.
